modules/troubleshooting-network-observability-flowlogs-pipeline-kafka.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-flowlogs-pipeline-kafka_{context}"]
+= Configuring flowlogs-pipeline to consume network flow from Kafka
```

Comment:
This lacks context. I'd say something like:

> If first deployed the `FlowCollector` with `deploymentModel: KAFKA` and deployed Kafka after it, the FlowCollector might not connect correctly to Kafka ([known bug](https://github.com/segmentio/kafka-go/issues/1044) in one of flowlogs-pipeline dependencies).

To make sure Kafka properly connects, you can restart the `FlowCollector`.

Reply:
Addressed